scalyr_agent/builtin_monitors/syslog_monitor.py

```diff
@@ -56,7 +54,10 @@
 from six.moves import range
 import six.moves.socketserver
 
-from scalyr_agent.builtin_monitors.thread_pool import ExecutorMixIn
+if six.PY2:
+    from scalyr_agent.builtin_monitors.thread_pool_dummy import ExecutorMixIn
+else:
+    from scalyr_agent.builtin_monitors.thread_pool import ExecutorMixIn
```

Comment:
There are aspects of thread_pool that aren't compatible with python 2?  I see the thread_pool_dummy doesn't do anything, it would be better if an equivalent approach could be implemented for python 2

Reply:
In python2 there's ThreadPool with a different API.
I was considering implementing the python2 solution around it, but then decided no to touch a code that already works, especially the python2 part which will be deprecated soon.
The ExecutorMixin from dummy package inherits ThreadingMixin - this is the way it was implemented before, it starts a new thread for every incoming request.